onnxruntime/test/onnx/main.cc

```diff
@@ -563,6 +564,7 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
 
 #ifdef USE_CUDA
   broken_tests.insert({"mxnet_arcface", "result mismatch"});
+  broken_tests.insert({"mask_rcnn_keras", "result mismatch"});
```

Comment:
Are we going to investigate this mismatch?

Reply:
Sure. We'll discuss it in today's scrum.